data/core/encyclopedia/geography.cfg

```diff
@@ -76,29 +101,36 @@ The central island of the archipelago is also called ‘Morogor’."
+
+" +
+        #po: Geography of Wesnoth
+        _ "<header>text='Notable cities:'</header>
+               • Weldyn: The capital of Wesnoth.
+               • Aldril: City lying on the Bay of Pearls.
+               • Blackwater Port: City lying south of the Bay of Pearls.
+               • Carcyn: Located between the Grey Woods and the Great River.
+               • Dan’Tonk: Wesnoth’s largest city, located in the center of the country, just west and north of Weldyn.
+               • Soradoc: The northernmost border outpost of Wesnoth, controls the confluence of the Weldyn River and the Great River.
+               • Fort Tahn: The southernmost border outpost, controls the north/south road crossing the River Aethen.
+               • Tath: Important fort city north of Dan’Tonk, exerts control over the wilderness country around the east of the Brown Hills and north to the Ford of Abez.
+               • Westin: An small yet important town in Kerlath, a province of Southern Wesnoth. The old Citadel of Westin can be found there." + "
+
+" +
+        #po: Geography of Wesnoth
+        _ "<header>text='Notable land features:'</header>
+               • Gryphon Mountain: Home of the fabled Gryphons.
+               • Ford of Abez: Shallow part of the Great River, it is usually controlled by Wesnothian forces.
+               • Weldyn River: It branches from the Great River and goes south.
+               • Great Central Plain: Area bounded by Weldyn, Dan’Tonk, and Fort Tahn, this plain is Wesnoth’s bread basket and home to most of its population.
+               • Dulatus Hills: These rolling hills bordering the Great Central Plain provide much of Wesnoth’s livestock and agriculture.
+               • Brown Hills: Wasteland surrounding Gryphon Mountain that is not well-populated and occasionally very dangerous.
+               • Horse Plains: Region of rolling plains just south of the Great River, bounded by Glyn’s Forest to the west and the River Weldyn to the east; the southern reach merges into the Central Plain. Home of the powerful Clans; the best horses in Wesnoth are bred here.
+               • Estmark Hills: Largish range rising south of the Great River and east of the Weldyn River. The northernmost portion, nearest the River Weldyn, has at various times been settled by Wesnothians, but the Kingdom’s control is tenuous at best and banditry is common.
+               • Glyn’s Forest: Sometimes known as the Royal Forest, named for one of Haldric II’s sons.
+               • Grey Woods: Large forest in the heart of the wilds of Wesnoth, located between Carcyn and Aldril and generally considered to be haunted.
+               • Isle of Alduin: An important island off the coast of Wesnoth home to the Great Academy of Magic, where many of Wesnoth's greatest mages were trained.
```

Comment:
Another straight quote.

Reply:
I can’t seem to find the striaght quote here?